python/tvm/relay/op/tensor.py

```diff
@@ -75,12 +74,25 @@ def add(lhs, rhs):
     -------
     result : relay.Expr
         The computed result.
+
+    Examples
+    --------
+    .. code:: python
```

Comment:
This code is yet to be runnable.

Reply:
When one is trying to write a deep learning model, and looking up the API docs, there are several steps we are typically going through.

Step 1. Check ndim of input and output of the API.
Step 2. Check shape of input and output of the API.
Step 3. Glance through some notes of the document.
Step 3. Very few practitioners try to check the content of output manually, because everyone knows what a `conv2d` does, or `dense` does.

IMHO, to be more intuitive, I believe it could be great to mark the shape of inputs and outputs of the example code inside the example, even if it is not runnable. The content of an NDArray is not that important relatively.

Then, the reason that I suggest some day we should make our example runnable is that when going through an API doc, many practitioners open an IPython notebook, or iTorch many many years ago. Everyone loves copy-pasting like me :-)

Reply:
Agreed, can you suggest an alternative that we can change it to?

Reply:
@tqchen I guess the simplest way is to add extra comments like
```
a = [...] # shape is [xxx]
b = [...] # shape is [yyy]
c = some_op(a, b) # shape is [zzz]
```

Ideally, a better way is to write actual code, and let CI runs it every time.